platform/ios/src/MGLMapView.h

```diff
@@ -597,7 +597,8 @@ MGL_EXPORT IB_DESIGNABLE
  * If the value of this property is smaller than that of the
  * minimumZoomLevel property, the behavior is undefined.
  *
- * The default maximumZoomLevel is 20.
+ * The default maximumZoomLevel is 22. The upper bound for this property
```

Comment:
Property names should be in backticks to benefit from jazzy’s autolinking feature. However, let’s say “default value of this property” here for consistency.